Helm elk updates

karun-singh opened the pull request:
- Added sample secrets directory
- Added script to generate secrets
- Updated README

abhi4578 reviewed (changes requested):
Seperate resource values (memory, cpu, storageclass, node-selector) for elastic, kibana and logstash

karun-singh commented:
- Separated resource-values for elasticsearch, logstash, and kibana. And added sample resource-values files.
- es-install.sh script now deploys the es-autoscaler-cron job for es-data-node autoscaling
- Updated README
